lib/common.sh

```diff
@@ -124,6 +126,14 @@ bd_() { #{{{
   echo "${tp_bold}$@${tp_reset}"
 } #}}}
 
+rd_() { #{{{
+  echo "${tp_red}$@${tp_reset}"
+} #}}}
+
+grn_() { #{{{
+  echo "${tp_green}$@${tp_reset}"  
+} #}}}
+
 cleanup_namespace() { #{{{
   unset -f dotbundle get_fullpath path_without_home path_without_dotdir
   unset -f __confirm prmpt bd_ dot_usage parse_linkfiles $0
```

Comment:
Add function names to avoid namespace pollution.

Reply:
Sorry for my late response, as well! Added both functions to `cleanup_namespace()`.